tests/Validator/ArrayListTest.php

```diff
@@ -23,7 +23,7 @@ public function testCanValidateNumericValues(): void
     {
         $arrayList = new ArrayList(new Numeric());
         $this->assertTrue($arrayList->isValid([1, 2, 3]));
-        $this->assertFalse($arrayList->isValid(1, '2', 3));
+        $this->assertFalse($arrayList->isValid(1));
```

Comment:
Why was this test changed?

Reply:
PhpStan used to say that it needs one parameter, but 3 parameters are passed. Let me know how you want me to proceed with this. Reverting the code back will require to add a line for PhpStan to ignore this line.